### ISSUE TITLE
[Bug] Blueprint not spawning (likely left/right corner related)

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#4554]
### Short Description of the issue:
A few of my older blueprints isn't spawning, but I think it's because they have some of the old left/right corners on them from back when they were in the game.

I can go back to the legacy voxel branch and they spawn fine, but re-exporting them in that branch and trying in the current branch results in a failed spawn. This is unusual as I had some other blueprints that were old enough to not spawn in current default, but this method worked to bring them up to a modern save format

### Steps to reproduce the issue:
- Try and load the attached bp

### Environment
- Starship EVO version: 22w06b
- Graphics Card: RX580

### Additional information:
[LeftRightCornerBug.zip](https://github.com/tsunamayo/Starship-EVO/files/8028578/LeftRightCornerBug.zip)

**Comment by tsunamayo** [comment]
@ProPeach Hi, well how big this ship is supposed to be? It seems that the save is tiny, so I guess at this point there is nothing I can (unless it is supposed to be tiny).
Can you send me the original blueprint?  
Thanks

**Comment by ProPeach** [comment]
So it seems like this issue might have been resolved -
It was my bad on the tiny save file, it seemed I tried to load the blueprint, saw it didn't load then exported it from the codex menu with a different name even though it hadn't loaded in front of me.
I was able to get the blueprint to load by spawning it in pre-neovox branch, exporting it and then spawning it in the Default branch. Although the left/right corner bricks were there in the pre-neovox branch, but disappeared in Default I guess because they're no longer in the Codex.

So the error seemed to be caused by too big of a jump between versions for an older blueprint to cross, solved by jumping in smaller intervals
